package.json

```diff
@@ -6,6 +6,9 @@
   "module": "./dist/esm5/index.js",
   "es2015": "./dist/esm/index.js",
   "types": "./dist/types/index.d.ts",
+  "typesVersions": {
+    ">=4.0": { "*": [ "dist/types/*" ] }
```

Comment:
question: Could we use this to express that the minimum supported version is 4.2? I mean, we're not going to support anything older than that.

Reply:
Probably?